src/bounce_solution/bounce_solution.cpp

```diff
@@ -0,0 +1,1256 @@
+// SPDX-FileCopyrightText: 2024 Lisa Biermann, Margarete Mühlleitner, Rui
+// Santos, João Viana
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+/**
+ * @file bounce solution calculator class
+ */
+
+#include <BSMPT/bounce_solution/bounce_solution.h>
+
+namespace BSMPT
+{
+
+/**
+ * @brief Default constructor
+ *
+ */
+BounceSolution::BounceSolution()
```

Comment:
Do you need the default ctr?

Reply:
No. Must have been used in the past for testing. Removed.